iOS/Airbnb_iOS/Airbnb_iOS/View/CollectionHeaderView.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  CollectionHeaderView.swift
+//  Airbnb_iOS
+//
+//  Created by 김한솔 on 2022/05/24.
+//
+
+import UIKit
+
+final class CollectionHeaderView: UICollectionReusableView {
+
+    static let identifier = "HeaderView"
+
+    private lazy var label: UILabel = {
+        let label = UILabel()
+        label.textColor = .black
+        label.textAlignment = .left
+        label.numberOfLines = 0
+        label.translatesAutoresizingMaskIntoConstraints = false
+        self.addSubview(label)
+        return label
+    }()
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+
+        backgroundColor = .clear
+        setLayout()
+    }
+
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        fatalError()
+    }
+
+    override func prepareForReuse() {
+        super.prepareForReuse()
+        setHeaderText(text: nil)
+    }
+
+    func setHeaderText(text: String?) {
+        label.text = text
+    }
+
+    func setHeaderFontSize(size: CGFloat) {
+        label.font = .systemFont(ofSize: size, weight: .init(rawValue: 400))
+    }
+}
+
+private extension CollectionHeaderView {
+
+    func setLayout() {
+        NSLayoutConstraint.activate([
+            label.centerYAnchor.constraint(equalTo: centerYAnchor),
+            label.leadingAnchor.constraint(equalTo: leadingAnchor, constant: 16),
+            label.trailingAnchor.constraint(equalTo: trailingAnchor, constant: 16)
```

Comment:
```suggestion
            // trailing 조건이 -16이 되어야 좌우 여백이 동일하게 처리됩니다🙂
            label.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -16)
```

AutoLayout 어렵지 않지만 생소해서 학습 기간동안 아래 영상 보면서 정리하면 좋지 않을까 싶어요🙂

* [인프런의 iOS AutoLayout을 활용한 실전 UI구성 전략 - 카카오톡 같은 고급 UI 만들기](https://www.inflearn.com/course/autolayout-ui_ios#curriculum) 
* WWDC Video가 나온지 시간이 조금 되었지만, AutoLayout 영상

Reply:
안그래도 오토레이아웃에 관해 부족함이 있어 조금 더 자료를 찾아보면서 공부하려고 계획했었는데 추천 감사합니다!
해당 부분 말씀처럼 수정하고, 오토레이아웃 빡세게 공부해보겠습니다! 감사합니다!